CLAUDE.md

```
@@ -17,23 +17,21 @@ Flake-based: `pyproject.toml` is authoritative for deps. No imperative installs.
 
 ### Branch Model
 
-All work goes through **feature branches** that are **squash-merged** onto `history/rewrite` via GitHub PRs.
+All work goes through **feature branches** that are **squash-merged** onto `master` via GitHub PRs.
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)